language/stdlib/modules/LibraVersion.move

```diff
@@ -41,19 +50,35 @@ module LibraVersion {
         );
 
         LibraConfig::set<LibraVersion>(
-            account,
+            lr_account,
             LibraVersion { major }
         );
     }
+    spec fun set {
+        include LibraTimestamp::AbortsIfNotOperating;
+        include Roles::AbortsIfNotLibraRoot{account: lr_account}; // Aborts if not LibraRoot [B19].
+        aborts_if LibraConfig::get<LibraVersion>().major >= major with Errors::INVALID_ARGUMENT;
+        include LibraConfig::SetAbortsIf<LibraVersion>{account: lr_account};
+        include LibraConfig::SetEnsures<LibraVersion>{payload: LibraVersion { major }};
+    }
 
     spec module {
         /// After genesis, version is published.
         invariant [global] LibraTimestamp::is_operating() ==> LibraConfig::spec_is_published<LibraVersion>();
 
-        /// The permission "UpdateLibraProtocolVersion" is granted to LibraRoot [B20].
-        invariant [global, isolated] forall addr: address where exists<LibraVersion>(addr):
+        /// The permission "UpdateLibraProtocolVersion" is granted to LibraRoot [B19].
+        invariant [global, isolated] forall addr: address where exists<LibraConfig<LibraVersion>>(addr): //  TODO: Is this a valid invariant although it is related to another module (e.g., LibraConfig).
             addr == CoreAddresses::LIBRA_ROOT_ADDRESS();
     }
-}
 
+    /// Only "set" can modify the LibraVersion config [B19]
```

Comment:
This schema apply illustrates that schema/apply has the indeed the same "problem" as global invariants. As this is applied only to post conditions of this module, it does not enforce anything about others outside of this module making changes the `LibraConfig<LibraVersion>`. 

@shazqadeer